src/pages/patientView/trialMatch/TrialMatchTable.tsx

```diff
@@ -0,0 +1,236 @@
+                                        <If condition={cgIndex < armMatch.matches.length - 1}><hr className={styles.criteriaHr}/></If>
+                                    </span>
+                                ))}
+                            </div>
+                             <If condition={armMatch.armDescription !== ''}>
+                                <div className={styles.armDiv}>
+                                    <span>Arm: {armMatch.armDescription}</span>
+                                </div>
+                            </If>
+                            <If condition={armMatch.drugs.length > 0}>
+                                <div className={styles.armDiv}>
+                                    <span>Intervention: {armMatch.drugs.join(', ')}</span>
+                                </div>
+                            </If>
+                        </div>
+                        <If condition={index < trial.matches.length - 1}><hr className={styles.criteriaHr}/></If>
+                    </div>
+                ))}
+            </div>
+        ),
+        width: this.columnsWidth[ColumnKey.MATCHING_CRITERIA]
+    }];
+
+    public getClinicalMatch(clinicalGroupMatch: IClinicalGroupMatch) {
+        return (
+            <span className={styles.firstRight}>
+                <span className={styles.secondLeft}>{clinicalGroupMatch.trialAgeNumerical + ' yrs old'}</span>
+                <span className={styles.secondRight}>
+                    {clinicalGroupMatch.trialOncotreePrimaryDiagnosis.positive.join(', ')}
+                    {clinicalGroupMatch.trialOncotreePrimaryDiagnosis.negative.length > 0 &&
+                        <span>
+                            <b> except in </b>
+                            { clinicalGroupMatch.trialOncotreePrimaryDiagnosis.negative.length < 4 ? (
+                                <span>{clinicalGroupMatch.trialOncotreePrimaryDiagnosis.negative.join(', ').replace(/,(?!.*,)/gmi, ' and')}</span>
+                            ) : (
+                                <DefaultTooltip
+                                    placement='bottomLeft'
+                                    trigger={['hover', 'focus']}
+                                    overlay={this.tooltipClinicalContent(clinicalGroupMatch.trialOncotreePrimaryDiagnosis.negative)}
+                                    destroyTooltipOnHide={false}
+                                    onPopupAlign={placeArrowBottomLeft}
+                                >
+                                    {this.mainContent(clinicalGroupMatch.trialOncotreePrimaryDiagnosis.negative.length + ` cancer types`)}
+                                </DefaultTooltip>
+                                )}
+                        </span>
+                    }
+                </span>
+            </span>
+        );
+    }
+
+    public getGenomicMatch(clinicalGroupMatch: IClinicalGroupMatch) {
+        return (
+            <span className={styles.firstLeft}>
```

Comment:
insetad of attaching classes, you could probably use css selectors first-child, last-child

Reply:
This is not for first child. This is for flex-box tricks. Maybe the name I assigned is not so clear. https://css-tricks.com/snippets/css/a-guide-to-flexbox/